include/api.h

```diff
@@ -40,6 +40,7 @@
 #endif /* __cplusplus */
 
 #include "pros/adi.h"
+#include "pros/aivision.h"
```

Comment:
So I'm not really a big fan of the "aivision" name, at least make it `ai_vision` or `ai_camera` (underscore matters the most, vision might be a little to ambigious)?

Reply:
(actually vision is fine but I'd prefer a underscore between ai and vision just for readability's sake)